src/main/groovy/com/ullink/BaseNuGet.groovy

```diff
@@ -33,7 +33,15 @@ public class BaseNuGet extends Exec {
         if (!localNuget.exists()) {
```

Comment:
Note that this folder/cache has the version in its name, hence it'll never update once downloaded.
(because the first time it'll download the latest version and save it in 'latest' folder, and never re-download it again)

You probably need to either:
- expire this cache at some point
- always download the exe when 'latest' is set (no cache)
- query nuget for which version is actually latest
...